### PR TITLE
Creating SSO 7.6 Release Notes

### LINKED ISSUE #1593: Create Release Notes for SSO 7.6
### Description

Add new features and changes to Tech Preview

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andymunro**
Closes #1593

https://github.com/miquelsi Can you approve this?

### CODE REVIEW

File: release_notes/topics/product/7_6.adoc
Comment (vmuzikar) [inline release_notes/topics/product/7_6.adoc]:
```suggestion
* The `podDisruptionBudget` field in the Keycloak CR is deprecated and will be ignored when the Operator is deployed on OpenShift 4.12 and higher. As a workaround, see the link:{upgradingguide_link}#rh_sso_7_6[{upgradingguide_name}].
```

**Review by vmuzikar — CHANGES_REQUESTED** [review]

**Review by sguilhen — DISMISSED** [review]
Structurally it looks to me. As for the contents of the release notes, I'll leave it to @pskopek to verify if they really reflect what is being added to 7.6.

**Review by vmuzikar — DISMISSED** [review]
From the Operator PoV, LGTM. Thanks @andymunro!

**Review by abstractj — DISMISSED** [review]

**Comment by abstractj** [comment]
@andymunro I can merge after @pskopek's review.

File: topics/templates/document-attributes-product.adoc
Comment (miquelsi) [inline topics/templates/document-attributes-product.adoc]: 18.0.0

**Comment by andymunro** [comment]
Thanks, @miquelsi Please check again and approve if okay.

**Review by vmuzikar — APPROVED** [review]

File: release_notes/master-docinfo.xml
Comment (miquelsi) [inline release_notes/master-docinfo.xml:13]: Shouldn't we use {cdate} here, as you did in other places?

Comment (andymunro) [reply release_notes/master-docinfo.xml:13]: > Shouldn't we use {cdate} here, as you did in other places?

Yes. This is already duplicated in downstream and merged.  I am trying to make sure the upstream and downstream always match.  I will fix it next time, because it is working now.

**Pull request merged**
